config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits.yaml

```diff
@@ -63,3 +63,44 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-gcp
       testgrid-tab-name: pr-make
+    # conformance test against kubernetes master branch with `kind`, skipping
+    # serial tests so it runs in ~20m
+  - name: pull-cluster-api-provider-gcp-make-conformance
```

Comment:
Should we make this a postsubmit job instead?

Reply:
yes, once i get it to work :)